guide/16-geoenrichment-guides/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>"Which zip code should the company pick? The median age for all 3 zip codes is fairly low and we can <u>say that these zip codes are still pretty young</u>. So, <u>depending on the type of healthcare facility this company wants to open</u>, these zip codes may not be right fit. "</p><ol><li>Mohit, can we rephrase the two underlined sentences here?</li><li>I might have missed it somewhere, but did we mention the purpose of this company opening health case facilities is to address diseases or sickness mostly seen on elderly somewhere? Not sure if it is safe to assume older people would be better fitting reasons to open clinics...</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I rephared the sentence. For some reason, I cannot see the sentences you have underlined. Please review and let me know if it needs correction.</p>